src/components/PullRequestList/index.tsx

```diff
@@ -1,11 +1,11 @@
-import { FC, useState, useEffect } from 'react';
+import React, { FC, useState, useEffect } from 'react';
 import Head from '@/components/head';
 import Layout from '@/components/Layout';
 import PullRequest from '@/components/pullRequests';
-import CardShimmer from '@/components/Loaders/cardShimmer';
 import axios from 'axios';
 import styles from './PullRequestList.module.scss';
 
+axios.defaults.timeout = 3000;
```

Comment:
Use our custom hook for making requests.
https://github.com/Real-Dev-Squad/website-status/blob/develop/src/hooks/useFetch.ts

Reply:
Used axios instead of custom fetch because there are multiple hooks to set empty array in case of error and detecting bottom.
Was not able to do the same with custom hooks.

Reply:
Is there any way you can make the custom hook usable in this usecase?
If yes let me know, we'll create an issue for making it more configurable.

Reply:
Yes, I think I can make it using custom fetch and A new issue would be better,then doing it in this PR.

Reply:
Can you create the issue, explain what changes you want.

Reply:
Ok, So this PR will be merged?

Reply:
created new issue.Please check

Reply:
Can you make use of the `useFetch` hook as shown in the below example?
```js
  const {
    response,
    error,
    isLoading,
  } = useFetch(OPEN_PULL_REQUEST_URL);
  
  useEffect(() => {
    // Handle error
    // Empty array
  }, [error])
```

Reply:
> Please ignore above suggestion, I think it won't work for making requests without a refresh / remount

We can use the `useFetch` to make a request only at the mounting phase of the component.
You'll need to use our custom `fetch` function.
Let me know if I am missing something.